src/components/NoteWhiteArea.scss

```diff
@@ -0,0 +1,8 @@
+$shadow-color1: #bbc2c7;
+$shadow-color2: #eff49f;
```

Comment:
These should go in `src/styles/colors.scss` and following the double-naming-scheme used there. Calling them 1 and 2 is not enough.